spec/facebook/messenger/subscriptions_spec.rb

```diff
@@ -55,7 +57,7 @@
                 'fbtrace_id' => 'Hlssg2aiVlN'
               }
             ),
-            status: 200,
+            status: 400,
```

Comment:
It makes more sense if Facebook responds with `HTTP 400 Bad Request` for this request, being an error and all, but are you sure that it does now?